routing/dht/routing.go

```diff
@@ -84,7 +84,10 @@ func (dht *IpfsDHT) GetValue(ctx context.Context, key key.Key) ([]byte, error) {
 	ctx, cancel := context.WithTimeout(ctx, time.Minute)
 	defer cancel()
 
-	vals, err := dht.GetValues(ctx, key, 16)
+	// retrieve a majority of the expected record count
+	majority := (KValue / 2) + 1
```

Comment:
well, looks like i had the wrong number in my head for K, its 10, not 20. That explains a lot.